examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>If you are open to suggestion, consider a heading like "## Next Steps" or "## Where to From Here" like https://github.com/NVDLI/notebooks/blob/master/even-easier-cuda/An_Even_Easier_Introduction_to_CUDA.ipynb</p><p> </p><p>Maybe also identify the suggested notebook by name since it's a somewhat canned experience:</p><p> </p><p>"02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset shows how to use Merlin Models to explore a variety of recommender models with your dataset."</p><p> </p><p>+1 for providing guidance on the next step.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I added the title. I did not want to link to a notebook, as we do not have integration tests, etc. for links. I am concerned that we will rename notebooks, etc. and then these links will break</p>